Remove embedded to oneToOne, add new Migrations (#155)
Sylius/RefundPlugin: docs/improvements.md

> ### Improvements

Reviewer: There is no reference to this file anywhere

> 1. Removed ``CreditMemoChannel`` and replaced it to ``Sylius\Component\Core\Model\ChannelInterface``

Reviewer: ```suggestion
1. Removed ``CreditMemoChannel`` and replaced by ``Sylius\Component\Core\Model\ChannelInterface``.
```

> 2. Replaced  ``CustomerBillingData``, ``ShopBillingData`` embedded to association and added interfaces ``CustomerBillingDataInterface``, ``ShopBillingDataInterface``

Reviewer: Sth like:
```suggestion
2. Replaced  ``CustomerBillingData`` and ``ShopBillingData`` value objects by entities with ``CustomerBillingDataInterface`` and ``ShopBillingDataInterface`` interfaces.
```
